sympy/polys/rings.py

```diff
@@ -2958,6 +2959,219 @@ def subresultants(self, g, x=None):
 
         return R
 
+    def sparse_free_sym(self):
+        """
+        Find the smallest index of the free symbolic variable in a given
+        polynomial.
```

Comment:
A sparse polynomial method does not need to have "sparse" in the method name.

I think usually this is what would be referred to as a "main variable" in textbooks etc.